src/imitation/scripts/common/rl.py

```diff
@@ -42,6 +53,28 @@ def fast():
     locals()  # quieten flake8
 
 
+@rl_ingredient.named_config
+def ppo():
+    # For recommended PPO hyperparams in each environment, see:
+    # https://github.com/DLR-RM/rl-baselines3-zoo/blob/master/hyperparams/ppo.yml
+    rl_cls = stable_baselines3.PPO
+    locals()  # quieten flake8
+
+
+# TODO(yawen): The current implementation of Soft Actor Critic in SB3 only supports
```

Comment:
This isn't really a TODO for imitation, I'd just add a one-line comment warning that SAC is currently continuous only. (And maybe link to that issue.)

Reply:
You're right.